entry/src/main/java/com/thanosfisherman/mayi/MainAbility.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thanosfisherman.mayi;
+
+import ohos.aafwk.ability.Ability;
+import ohos.aafwk.content.Intent;
+import com.thanosfisherman.mayi.slice.MainAbilitySlice;
+
+public class MainAbility extends Ability {
+
+    @Override
+    public void onStart(Intent intent) {
+        super.onStart(intent);
+        super.setMainRoute(MainAbilitySlice.class.getName());
+
```

Comment:
Please remove the white spaces to avoid lint issues.

Reply:
Done